src/pages/core/external-plugins/app-data.md

```diff
@@ -0,0 +1,250 @@
+---
+title: AppData Plugin
+metaTitle: Core - AppData Plugin
+description: Learn about the MPL Core AppData Plugin
+---
+
+## What is an AppData Plugin?
+
+The `AppData` third party plugin stores and contains arbitrary data that can be written to by the `dataAuthority`. Note this is different then the overall plugin authority stored in the `ExternalRegistryRecord` as it cannot update/revoke authority or change other metadata for the plugin.
```

Comment:
Nit: external plugin instead of "third party"

Reply:
Think I copy pasta'd that from your code comments, so might need changing there too.